src/controllers/translation.ts

```diff
@@ -19,9 +15,18 @@ export interface Translation {
   translation: string;
 }
 
-// Due to limit on inputs used to train the model, the maximum
-// Igbo translation input is 120 characters
-const IGBO_ENGLISH_TRANSLATION_INPUT_MAX_LENGTH = 120;
+// TODO: move this information to remote config
+const SUPPORTED_TRANSLATIONS = {
```

Comment:
I think we would want to update this map to rely on nested maps for better indexing:

```ts
const SUPPORTED_TRANSLATIONS = {
  [LanguageEnum.IGBO]: {
    [LanguageEnum.ENGLISH]: {
      maxInputLength: 120,
      translationAPI: IGBO_TO_ENGLISH_API,
    }
  },
  [LanguageEnum.ENGLISH]: {
    [LanguageEnum.IGBO]: {
      maxInputLength: 120,
      translationAPI: ENGLISH_TO_IGBO_API,
    }
  }
}
```

So then when we use `SUPPORT_TRANSLATIONS` we can index using our `LanguageEnum` values:

```ts
const translation = SUPPORTED_TRANSLATIONS[requestBody.sourceLanguageCode][requestBody.destinationLanguageCode] // === { maxInputLength: 120, translationAPI: IGBO_TO_ENGLISH_API }
```

Reply:
I'm also interested to hear why we would want to move this to RemoteConfig! I think it's fine having this object here, but I might be missing something

Reply:
I ended up getting something to work that typescript could actually be happy with by indexing all the languages on the first level of the object. Inner languages don't need to map out each one as not to be overly exhaustive.